app/_gateway_entities/target.md

```diff
@@ -11,22 +11,65 @@ tools:
     - deck
     - terraform
 
-description: A target identifies a specific instance of an upstream service.
+description: A Target is an IP address/hostname with a port that identifies an instance of a backend service.
+
+related_resources:
+  - text: Upstream entity
+    url: /gateway/entities/upstream/
+  - text: Routing in {{site.base_gateway}}
+    url: /gateway/routing/
+  - text: Proxying with {{site.base_gateway}}
+    url: /gateway/traffic-control/proxy/
 
 schema:
     api: gateway/admin-ee
     path: /schemas/Target
 
 ---
 
-## What is a target?
+## What is a Target?
+
+{{page.description}} Each [Upstream](/gateway/entities/upstream/) can have many Targets. Targets are used by Upstreams for load balancing. For example, if you have an `example_upstream` Upstream, you can point it to two different Targets: `httpbin.konghq.com` and `httpbun.com`. This is so that if one of the servers (like `httpbin.konghq.com`) is unavailable, it automatically detects the problem and routes all traffic to the working server (`httpbun.com`).
+
+The following diagram illustrates how Targets are used by Upstreams for load balancing:
+
+<!--vale off-->
 
-{{page.description}}
+{% mermaid %}
+flowchart LR
+  A("`Route 
+  (/mock)`")
+  B("`Service
+  (example_service)`")
+  C(Load balancer)
+  D(httpbin.konghq.com)
+  E(httpbun.com)
+  
+  subgraph id1 ["`**KONG GATEWAY**`"]
+    A --> B --> C
+  end
 
-## Use cases for targets
+  subgraph id2 ["`Targets (example_upstream)`"]
```

Comment:
```suggestion
  subgraph id2 ["`Targets`"]
```

This is a known [issue](https://github.com/mermaid-js/mermaid/issues/3806), I would say let's remove the extra text for now